src/Microsoft.DotNet.Build.Tasks/Targets/packages.targets

```diff
@@ -76,4 +80,47 @@
       <_TempPackageVersion />
     </PropertyGroup>
   </Target>
+
+  <Import Condition="Exists('$(LocalPublishPropsFile)')"  Project="$(LocalPublishPropsFile)"/>
+  
+  <!-- Publish the packages locally -->
+  <Target Name="LocalPackagePublish" Condition="'@(Packages)'!=''"
```

Comment:
I actually think that Packages may conflict with other things we should consider a mores specific name. Perhaps LocalPackages.